instrumentation/logback-classic-1.2/src/main/java/ch/qos/logback/classic/Logger_Instrumentation.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ *  * Copyright 2022 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package ch.qos.logback.classic;
+
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.NewField;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.WeaveAllConstructors;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.agent.instrumentation.logbackclassic12.AgentUtil;
+import org.slf4j.Marker;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Weave(originalName = "ch.qos.logback.classic.Logger", type = MatchType.ExactClass)
+public abstract class Logger_Instrumentation {
+    @NewField
+    public static AtomicBoolean instrumented = new AtomicBoolean(false);
```

Comment:
Can this be final?

Reply:
Probably, but I don't think it matters much in this case.